### PR TITLE
[ORGANISERS-SUMMIT-2024] Pretalx => DoD Schedule Auto Update

### PR DESCRIPTION
**Pull request by KrisBuytaert**
Schedule updates for 2024-organisers-summit

Auto Release from Pretalx to DoD Schedule

***Conference Hook System***

### CODE REVIEW

**Comment by somatorio** [comment]
The generator script have to be updated to remove the `"` from the filenames
https://github.com/devopsdays/devopsdays-web/actions/runs/9973650734/job/27559495466?pr=14361#step:4:17

File: content/events/2024-organisers-summit/speakers/daniel-"phrawzty"-maher.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+Title: "Daniel "phrawzty" Maher"
```
Comment (somatorio) [inline content/events/2024-organisers-summit/speakers/daniel-"phrawzty"-maher.md:2]: Hugo is not handling this well

**Comment by KrisBuytaert** [comment]
@somatorio  I`m trying to get it sanitized on Pretalx itselve ...  by removing the "Phrawzty" from @phrawzty  's name ..  :)

File: content/events/2024-organisers-summit/speakers/daniel--maher.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
```
Comment (somatorio) [inline content/events/2024-organisers-summit/speakers/daniel--maher.md:1]: Do we need 3 entries for Dan?

**Review by somatorio — CHANGES_REQUESTED** [review]

File: content/events/2024-organisers-summit/speakers/daniel--maher.md (hunk 2 above)
Comment (KrisBuytaert) [reply content/events/2024-organisers-summit/speakers/daniel--maher.md]: I`m trying to fix this manually .. then @toshywoshy  can fix the code later :)

File: content/events/2024-organisers-summit/speakers/daniel-"phrawzty"-maher.md (hunk 1 above)
Comment (toshywoshy) [reply content/events/2024-organisers-summit/speakers/daniel-"phrawzty"-maher.md]: OK, will add this to the TODO list to test and see how we can accomodate this

File: content/events/2024-organisers-summit/speakers/daniel--maher.md (hunk 2 above)
Comment (toshywoshy) [reply content/events/2024-organisers-summit/speakers/daniel--maher.md]: Yes, this is still a know bug, I have also added this to the TODO list
Currently rebase or resync and that should solve the problem

**Pull request closed without merging**